README.md

```diff
@@ -87,6 +89,12 @@ After adding a job, it will remain active and will run every time QuickBooks Web
 
 Use the [Onscreen Reference for Intuit Software Development Kits](https://developer-static.intuit.com/qbSDK-current/Common/newOSR/index.html) (use Format: qbXML) to see request and response formats to use in your jobs. Use underscored, lowercased versions of all tags (e.g. `customer_query_rq`, not `CustomerQueryRq`).
 
+### Referencing memory values when constructing requests ###
+
+A QBWC::Worker#requests method cannot access values that are in-memory (local variables, model attributes, etc.) at the time that QBWC.add_job is called; however, in lieu of using QBWC::Worker#requests, you can optionally construct and pass requests directly to QBWC.add_job (scalar request or array of requests). These requests will be immediately persisted by QBWC.add_job (in contrast to requests constructed by QBWC::Worker#requests, which are persisted during a QuickBooks Web Connector session).
```

Comment:
AFAICT, the reason to generate and pass requests to `QBWC.add_job` is precisely because variables aren't available to `QBWC::Worker#requests`.  This current verbiage was revised in response to [prior feedback](https://github.com/skryl/qbwc/pull/38#discussion_r22471673) in order to provide some motivation/rationale for generating requests in advance vs. using a worker.